### PR TITLE
feat(statics): add SOL tokens for FTX transfer drift

### PR DESCRIPTION
**Pull request by Vijay-Jagannathan**
Ticket: WIN-1405

### CODE REVIEW

**Comment by Vijay-Jagannathan** [comment]
Please merge if you're the last reviewer, ty!

**Review by ajays97 — APPROVED** [review]
Token and token configs (decimals, token address, name) looks good.

File: modules/statics/src/base.ts [hunk 1]
```diff
@@ -1758,6 +1758,8 @@ export enum UnderlyingAsset {
   'sol-perp' = 'sol-perp',
   'sol-woo' = 'sol-woo',
   'sol-weth' = 'sol-weth',
+  'btc-sollet' = 'btc-sollet',
+  'eth-sollet' = 'eth-sollet',
```
Comment (ajays97) [inline modules/statics/src/base.ts:1762]: Only these tokens to be onboarded for fixing FTX TDs? @Vijay-Jagannathan

Comment (Vijay-Jagannathan) [reply modules/statics/src/base.ts:1762]: We already added a few recently. This ticket deals only with a specific set of txns, which need these 2 tokens specifically

**Pull request merged**